packs/malevolence-bestiary/corrupted-nosoi.json

```diff
@@ -215,7 +215,18 @@
                 "requirements": {
                     "value": ""
                 },
-                "rules": [],
+                "rules": [
```

Comment:
No note?

Reply:
It's currently set up as attackEffects, so the note is implicit.